Client/Frontend/Settings/SettingsNavigationController.swift

```diff
@@ -24,6 +24,16 @@ class ThemedNavigationController: UINavigationController {
         modalPresentationStyle = .formSheet
         modalPresentationCapturesStatusBarAppearance = true
         applyTheme()
+        
+        let separator = UIView()
+        separator.backgroundColor = UIColor.theme.ecosia.barSeparator
```

Comment:
I think you need to call this again in applyTheme() otherwise this color will stick after theme change

Reply:
Check again 👍